src/index.js

```diff
@@ -0,0 +1,92 @@
+import './style.css';
+
+const pressEnter = document.getElementById('press-enter');
+const hoverText = document.getElementById('button-hover');
+
+const showToDo = document.getElementById('task-list');
+const myTask = document.getElementById('my-task');
+const form = document.getElementById('form');
+const error = document.getElementById('error');
+
+// Hover message when hover over enter icon
+pressEnter.addEventListener('mouseover', () => {
+  hoverText.style.display = 'block';
+});
+
+pressEnter.addEventListener('mouseout', () => {
+  hoverText.style.display = 'none';
+});
+
+// Local Storage
+class ToDo {
+  constructor() {
+    this.taskList = JSON.parse(localStorage.getItem('storage-task')) || [];
+  }
+
+  addTask(description) {
+    const updateTask = [
+      ...this.taskList,
+      { description },
+    ];
+    this.updateStorage(updateTask);
+  }
+
+  // removeTask(id) {
+  //   // const updateTask = this.taskList.filter((it) => it.id !== id);
+  //   this.updateStorage(updateTask);
+  // }
```

Comment:
- [ ] Good job working on this, but please kindly remove the code comments as it pollutes the codebase.